swift/ios-shoppe-demo/Views/CheckBoxButtonCell.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CheckBoxButtonCell.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 5/28/20.
+//  Copyright © 2020 Harold Davis Jr. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CheckBoxButtonCell: UITableViewCell {
+    @IBOutlet weak var checkboxButton: UIButton!
+    var boxIsChecked: Bool = false
+    var checkoutView: CheckoutTableViewController?
+
+    @IBAction func checkbox(_ sender: Any) {
+        boxIsChecked = boxIsChecked ? false : true
```

Comment:
nit: maybe `boxIsChecked = !boxIsChecked` would be more readable?

Reply:
I agree :)